src/styles/table.scss

```diff
@@ -1,6 +1,9 @@
 /* Basic styling for generic markdown tables which don't currently
  * use Amplify UI primitives/classNames */
 table:not([class]) {
+  /* Add display: block so that we can add an overflow-x to horizontally scroll tables that are too wide */
+  display: block;
```

Comment:
I think this is fiiiiiine for now to fix the ugliness of the tables currently but I created a task for us to look into making tables scrollable while maintaining keyboard scrollability.

Basically, we need to wrap the table itself in a scrollable container and provide it some attributes to make it focusable so that someone navigating with keyboard alone can focus that container and use left/right keyboard arrows to scroll the container. There is a very basic example here (though it does not have focus styles) https://ui.docs.amplify.aws/react/components/scrollview#accessibility